lib/greek_stemmer.rb

```diff
@@ -21,6 +21,8 @@ def load_settings(key)
     end
   end
 
+  # Top-level stemming exceptions
+  STEP_0_EXCEPTIONS = load_settings("step_0_exceptions")
```

Comment:
Same as before, maybe hide this under a memoized method instead of using a constant?